lib/widgets/compose_box.dart

```diff
@@ -999,8 +999,21 @@ class _StreamComposeBoxState extends State<_StreamComposeBox> implements Compose
     super.dispose();
   }
 
+  Widget? _errorBanner(BuildContext context) {
```

Comment:
There's already an `_errorBanner` method, in `_FixedDestinationComposeBoxState`, and it duplicates a lot of this logic. Can we centralize the computation, perhaps in an early-return style in the build method of `ComposeBox`?